CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 5.0.0 - 2021-06-04
```

Comment:
I just noticed you changed the version to rc1 — should this header be changed too?

Reply:
Oosp, I will do that ASAP. Thanks for taking a look at this PR.